docs/source/release_notes/lts/next.rst

```diff
@@ -88,6 +88,118 @@ introduced by the next LTS version are:
+  virtual environments (in addition to installations *inside* virtual
+  environments).
+
+
+* A :ref:`new plugin <glib-plugin>` enables users to list and execute
+  tests based on the `GLib test framework
+  <https://developer.gnome.org/glib/stable/glib-Testing.html>`_.  This
+  plugin allows individual tests inside a single binary to be listed
+  and executed.
+
+* Users of the YAML test loader have now access to a few special keys
+  that can tweak test attributes, including adding prefixes to test
+  names.  This allows users to easily differentiate among execution of
+  the same test, but executed different configurations.  For more
+  information, look for "special keys" in the :ref:`YAML Loader plugin
+  documentation <yaml_loader>`.
+
+* Users can now dump variants to a (JSON) file, and also reuse a
+  previously created file in their future jobs execution.  This allows
+  users to avoid recomputing the variants on every job, which might
+  bring significant speed ups in job execution or simply better
+  control of the variants used during a job.  Also notice that even
+  when users do not manually dump a variants file to a specific
+  location, Avocado will automatically save a suitable file at
+  ``jobdata/variants.json`` as part of a Job results directory
+  structure.
+
+* SIMPLE tests were limited to returning PASS, FAIL and WARN statuses.
+  Now SIMPLE tests can now also return SKIP status.  At the same time,
+  SIMPLE tests were previously limited in how they would flag a WARN
+  or SKIP from the underlying executable.  This is now configurable by
+  means of regular expressions.
+
+* The :mod:`avocado.utils.process` has seen a number of changes
+  related to how it handles data from the executed processes.  In a
+  nutshell, process output (on both ``stdout`` and ``stderr``) is now
+  considered binary data.  Users that need to deal with text instead,
+  should use the newly added
+  :attr:`avocado.utils.process.CmdResult.stdout_text` and
+  :attr:`avocado.utils.process.CmdResult.stderr_text`, which are
+  convenience properties that will attempt to decode the ``stdout`` or
+  ``stderr`` data into a string-like type using the encoding set, and
+  if none is set, falling back to the system default encoding.  This
+  change of behavior was needed to accommodate Python's 2 and Python's
+  3 differences in bytes and string-like types and handling.
+
+* The TAP result format plugin received improvements, including
+  support for reporting Avocado tests with CANCEL status as SKIP
+  (which is the closest status available in the TAP specification),
+  and providing more visible warning information in case Avocado tests
+  finish with WARN status (while maintaining the test as a PASS, since
+  TAP doesn't define a WARN status).
+
+* Redundant (and deprecated) fields in the test sections of the JSON
+  result output were removed.  Now, instead of ``url``, ``test`` and
+  ``id`` carrying the same information, only ``id`` remains.
```

Comment:
Would you please add following:

* d2458c6dc5121384bd78c0768b8f040ed037f328 - new arguments
* bc59f8506fc4700a111d66ac3a6ab1024f4e8c25 - removed 36 compatibility
* 4eed345e4bd59993e03c3e2d21098ee9f0ea5a57 - new functions in multipath
* 9b9a335c3fe548348deffe15fe528ac4b0e2a3a9 - subtest filter for dirs
* 27661323f98e4fb8172f39b9359ae567ce11a003 - subtest filter for simpletests
* b38129958c0c21c0c725b8cea021dc06398d6b71 - mux interface moved from `core` to `optional_plugins`
* 13316b0059c9dff1634c6f062f3836f5cab530de - split varianter and parameters modules
* python3 support
* `avocado variants` deprecated arguments removed
* 73c57528b628297e6a45c4d4d614bbddb39aa8a7 - stripped and define the test state content
* 51e46987af536048b55853ffdbfbde62b62cb822 - many internal renames from implementation to concept (`mux` -> `variants`)
* 2be50b1d6fd85c61d6592ac3289b4408e8b426e9 - find free port randomly
* 5bd8263644835acca85a498af0db5de8efe0208a - ordered tests discovery
* 35f2ec8af42837da22e7f54c612c5c7803fcfecc - wheel packages support
* 29449cc9319189b211c935812b68ac71ac552ad4 - different success-rate in html plugin
* 1f296fac5af714d2008b5bb5ea934c7829317bbe - legacy interface dropped
* 3647cae545a82edeb618743b82156d9adc87ee72 - utils.kernel - support for other than 3.x versions
* d639baf30ca5db492d9b1e34a58c919d70c19bde - DummyLoader moved to optional plugins
* ae272b29c6a751cb850c727c39fa0823460e63c4, 3eab6f58f8c2f77ef36f75d2108ec41ea4be3da4 - refinements to `SimpleTest` and `ExternalRunner` to avoid using `name` for commands, because test name can be altered by plugins (adjust your custom loaders to avoid similar issues)
* d716b018a8069a2a57bbddb63d9afe01c24a21e9 -`varianter` supports `__len__`